packages/teleport/src/Console/DocumentNodes/useNodes.ts

```diff
@@ -186,3 +198,14 @@ export default function useNodes({ clusterId, id }: stores.DocumentNodes) {
     onLabelClick,
   };
 }
+
+// sort logins by making 'root' as the first in the list
+const sortLogins = (logins: string[]) => {
+  const noRoot = logins.sort().filter(l => l !== 'root');
+  if (noRoot.length === logins.length) {
+    return logins;
+  }
+
+  noRoot.unshift('root');
+  return noRoot;
+};
```

Comment:
```suggestion
const sortLogins = (logins: string[]) => {
  const noRoot = logins.filter(l => l !== 'root').sort();
  if (noRoot.length === logins.length) {
    return logins;
  }

  return ['root', ...noRoot];
};
```

Reply:
Thanks, looks much better. I used this on [packages/teleport/src/Nodes/useNodes.ts](https://github.com/gravitational/webapps/pull/873/files/c211f70a376b4f31adb555d0e3647e2f73ca9919#diff-7d4b0ce921483e33f08ef1df96185b0e604089b55a02e7eb3b2455d24a392eb4) and exported it to use on this file, addressing both this and the other comment (https://github.com/gravitational/webapps/pull/873/commits/6f67c0adec0efa1ca36acfccdac56e3a338c42b4).

Thanks!